src/partials/forms/forms.component.ts

```diff
@@ -0,0 +1,38 @@
+import { Component, OnInit, Inject } from '@angular/core';
+import { Location } from '@angular/common';
+import { Subject } from 'rxjs/Subject';
+
+let style = require('./forms.css');
+
+@Component({
+  selector: 'forms',
+  styles: [style.toString()],
```

Comment:
your styles will be encapsulated
so if we need to set some general styles for forms it will not work for yours

just require styles without set Component.styles